### PR TITLE
FOX-24684: Fix small issues

### PR DESCRIPTION
**Pull request by kbridbur**
New Index page
screen shot 2019-01-08 at 1 33 55 pm

### CODE REVIEW

File: source/models/transcript.rst [hunk 1]
```diff
@@ -43,9 +43,10 @@ confidence             double           confidence score of the provided value.
 
 When a job is submitted with option :ref:`skip_diarization <options-model>` set to ``false`` or is not specified, the transcript will have the following format:
 
+**Transcript Object**
```
Comment (kbridbur) [inline source/models/transcript.rst]: Not sure these titles are even necessary. I think we could just have the json follow

File: source/conf.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
 # The short X.Y version
 version = u''
 # The full version, including alpha/beta/rc tags
-release = u'1.0.0'
+release = u'1.0.1'
```
Comment (kennygea) [inline source/conf.py]: minor, not really necessary. If it messes with linking, I wou;d just leave it as 1.0.0

File: source/endpoints/transcript.rst [hunk 3]
```diff
@@ -7,7 +9,7 @@ Transcript
 
 Gets an :ref:`transcript <transcript-model>` by :ref:`job <job-model>` id
 
-Note: all GET transcript requests must include an ``Accept`` header that specifies the desired output format. 
+Note: all GET transcript requests must include an ``Accept`` header that specifies the desired output format. Either plain text or revs special json format
```
Comment (kennygea) [inline source/endpoints/transcript.rst]: minor should be `Rev.ai's`

File: source/endpoints/account.rst [hunk 4]
```diff
@@ -39,5 +41,7 @@ Code                   Description
 
                         {
                           "title": "Authorization has been denied for this request"
-                        }    
+                        }  
+
+                       Caused by an old or invalid API Token, try regenerating your token on your `settings page`_.
```
Comment (kennygea) [inline source/endpoints/account.rst]: these should be API Key to keep consistent with other terminology

File: source/quickstart.rst [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+  curl -X POST "https://api.rev.ai/revspeech/v1beta/jobs" -H "Authorization: Bearer <api_key>" -H "Content-Type: application/json" -d "{\"media_url\":\"https://support.rev.com/hc/en-us/article_attachments/200043975/FTC_Sample_1_-_Single.mp3\",\"metadata\":\"This is a sample submit jobs option\"}"
+
+You’ll receive a response like this:
+
+.. code:: javascript
+
+  {"id":"2028247","created_on":"2018-09-15T05:14:38.13","name":"sample.mp3","metadata":"This is a sample submit jobs option for multipart","status":"in_progress"}
+
+This id (in this case 2028247) will allow you to retrieve your transcript.
```
Comment (kennygea) [inline source/quickstart.rst]: formatting for `id` and `2028247` I would use \` to highlight them

**Pull request merged**